docs/rules/no-chained-assignments.md

```diff
@@ -0,0 +1,49 @@
+# Disallow Use of Chained Assignments (no-chained-assignments)
+
+Chaining the assignment of variables can lead to unexpected results or be difficult to read.
+
+```js
+var a = 1,
```

Comment:
This example is a little bit confusing - it looks like this rule disallows using a single declaration for multiple assignments. I think we're fine with the second example only.